### PR TITLE
Toast: convert Light DOM LESS to SCSS

### PR DESCRIPTION
**Pull request by lalithkarikelli**
JIRA: SURF-1886

### LGTM's
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/less/components.less [hunk 1]
```diff
@@ -42,5 +42,4 @@
 @import 'components/table/index';
 @import 'components/tabset/index';
 @import 'components/text-input/index';
-@import 'components/textarea/index';
-@import 'components/toast/index';
+@import 'components/textarea/index';
```
Comment (100stacks) [inline src/less/components.less]: Please add a newline after this statement.

File: src/scss/components/toast/_index.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
-@import (reference) 'vars';
+@import 'vars';
```
Comment (100stacks) [inline src/scss/components/toast/_index.scss]: Please use double quotes:

```scss
@import "vars";
```

File: src/less/components.less (hunk 1 above)
Comment (lalithkarikelli) [reply src/less/components.less]: Please check the changes in latest commit.

File: src/scss/components/toast/_index.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/toast/_index.scss]: Please check the changes in latest commit.

**Comment by CITguy** [comment]
Please squash down to 1 commit.

**Pull request merged**